examples/advanced_example.html

```diff
@@ -148,7 +148,8 @@
         markerClusterer = new MarkerClusterer(map, markers, {
           maxZoom: zoom,
           gridSize: size,
-          styles: styles[style]
+          styles: styles[style],
+          imagePath: '../images/m'
```

Comment:
My intention was to make the `imagePath` option explicit in the examples. If someone copies the example he/she will hopefully notice this line and adapt it to their setup. But I can scrap it if you don't agree..